src/js/components/graphs/thumbnail-components/rectangle.jsx

```diff
@@ -67,13 +67,13 @@ var Rectangle = React.createClass({
             <image
                 className={classes}
                 x={this.state.x }  width={this.state.r*2}
-                y={this.state.y }        height={this.state.r*2}
+                y={this.state.y }  height={this.state.r*2}
                 href={thumb}
                 preserveAspectRatio={"xMinYMin slice"}
                 clipPath={clipVal}
-                onClick={this.props.eventHandler.bind(null, this.props.data)}
-                onMouseEnter={this.props.onMouseEnterH.bind(null,this.props.data)}
-                onMouseLeave={this.props.onMouseLeaveH.bind(null,this.props.data)}
+                onClick={this.props.eventHandler?this.props.eventHandler.bind(null, this.props.data):null}
```

Comment:
Does attaching null to an event handler not throw an error ?

Not sure if this is the most ideal way to set this up.